string product_code
string parent_order_id
string parent_order_acceptance_id
string event_date
string event_type
string parent_order_type
string reason
string child_order_type
float32 parameter_index
string child_order_acceptance_id
string side
float32 price
float32 size
string expire_date